app/src/main/java/com/courseproject/tindar/ui/matchlist/MatchListFragment.java

```diff
@@ -1,37 +1,71 @@
 package com.courseproject.tindar.ui.matchlist;
 
+import android.content.Intent;
 import android.os.Bundle;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.TextView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
 
 import androidx.annotation.NonNull;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProvider;
 
-import com.courseproject.tindar.databinding.FragmentMatchListBinding;
+import com.courseproject.tindar.BlankNavViewModel;
+import com.courseproject.tindar.ChatActivity;
+import com.courseproject.tindar.R;
+import com.courseproject.tindar.controllers.likelist.LikeListController;
+import com.courseproject.tindar.ds.DatabaseHelper;
+import com.courseproject.tindar.usecases.likelist.LikeListInputBoundary;
+import com.courseproject.tindar.usecases.likelist.LikeListInteractor;
+import com.courseproject.tindar.usecases.likelist.LikeListDsGateway;
+import com.courseproject.tindar.usecases.likelist.LikeListResponseModel;
 
-public class MatchListFragment extends Fragment {
+public class MatchListFragment extends Fragment{
 
-    private FragmentMatchListBinding binding;
+    private String userId;
+    private ListView listView;
+    private LikeListController likeListController;
+    private String[] matchedUserIds;
```

Comment:
I guess it doesn't need to be dynamic, but I'd consider making this an ArrayList instead